changes/1195.deprecation.md

```diff
@@ -0,0 +1,2 @@
+`RESTClient.edit_permission_overwrites` deprecated
+ - Renamed to `RESTClient.edit_permission_overwrite`
```

Comment:
Let me know if you want this to be more specific or wording changed.

Reply:
Could just be simplified to: ```py
```suggestion
`RESTClient.edit_permission_overwrites` renamed to `RESTClient.edit_permission_overwrite`
```